ingest-app/src/cmr/ingest/services/granule_bulk_update/mime_type/umm_g.clj

```diff
@@ -0,0 +1,35 @@
+(ns cmr.ingest.services.granule-bulk-update.mime-type.umm-g
+  "Contains functions to update UMM-G granule metadata for MimeType granule bulk update."
+  (:require
+   [clojure.string :as string]
+   [cmr.common.services.errors :as errors]))
+
+(defn- transform-link
```

Comment:
Super nitpick that you can ignore: `transform` is a little vague. Something like `add-or-update-link` may be more descriptive, assuming you don't plan to add more functionality here